SpatialConnect/SpatialConnect.m

```diff
@@ -137,14 +137,14 @@ - (void)restartAllServices {
   [_serviceGraph restartAllServices];
 }
 
-- (void)connectBackend:(SCRemoteConfig *)r {
-  if (![_serviceGraph nodeById:[SCBackendService serviceId]]) {
-    _backendService = [[SCBackendService alloc] initWithRemoteConfig:r];
-    [self addService:_backendService];
-    [self startService:[SCBackendService serviceId]];
-  } else {
-    DDLogWarn(@"SCBackendService Already Connected");
-  }
+- (void)connectBackend:(id<SCBackendProtocol>)bp {
```

Comment:
Use of backend kind of gets confusing. Maybe its clearer to call it Server. `SpaconServer` & `ExchangeServer`

Reply:
I think we can leave it as `SCBackendProtocol` with `connectBackend` because you'll always see what backend you're connecting to....for example:
```
[sc connectBackend:[[SCExchangeBackend alloc] initWithRemoteConfig:c.remote]];```